libc-test/build.rs

```diff
@@ -2288,7 +2291,10 @@ fn test_android(target: &str) {
             // FIXME(android): bad function pointers:
             "isalnum" | "isalpha" | "iscntrl" | "isdigit" | "isgraph" | "islower" | "isprint"
             | "ispunct" | "isspace" | "isupper" | "isxdigit" | "isblank" | "tolower"
-            | "toupper" => true,
+            | "toupper" | "getpwuid_r" | "tmpfile" | "getpwnam" | "getpwuid" | "tcdrain"
+            | "cfgetispeed" | "cfgetospeed" | "cfsetispeed" | "cfsetospeed" | "tcgetattr"
+            | "tcsetattr" | "tcflow" | "tcflush" | "tcgetsid" | "tcsendbreak" | "cfmakeraw"
+            | "cfsetspeed" => true,
```

Comment:
These don't seem relevant to the change here

Reply:
What circumstances make these fail? We have Android tests in CI and as far as I know this hasn't been a problem

Reply:
I'm not sure, to be honest; I didn't spend the time to dig into the failures. It could be something specific to termux. 

I went ahead and removed these changes.